### PR TITLE
ADD:Lighthouse monitoring test (prometheus, prom-node-exporter and grafana)

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: controls/roles/manage-service/tasks/write-grafana-configuration.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+- name: Create directories for Grafana
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0655
+  loop:
+    - /opt/app/services/59827650-8ac6-11ec-81ec-f3dc0fb8f30e/grafana/provisioning/datasources
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/tasks/write-grafana-configuration.yml]: done!

File: controls/roles/manage-service/tasks/write-grafana-configuration.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  copy:
+    src: "{{ role_path }}/files/grafana/datasource.yml"
+    dest: /opt/app/services/59827650-8ac6-11ec-81ec-f3dc0fb8f30e/grafana/provisioning/datasources
+    mode: 0655
+  become: yes
+  when: stereum_service_configuration.command.find("GRAFANA_INI_LH") != -1
+
+- name: Copy Grafana's dashboard(s)
+  copy:
+    src: "{{ role_path }}/files/grafana/provisioning-lh/dashboards/BeaconProcessing.json"
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/tasks/write-grafana-configuration.yml:24]: done!

File: controls/roles/manage-service/tasks/write-grafana-configuration.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    mode: 0655
+  loop:
+    - "{{ grafana_config_path }}/provisioning/datasources"
+    - "{{ grafana_config_path }}/provisioning/dashboards"
+  become: yes
+
+- name: Copy Grafana's "datasource" file
+  copy:
+    src: "{{ role_path }}/files/grafana/datasource.yml"
+    dest: "{{ grafana_config_path }}/provisioning/datasources"
```
Comment (stefa2k) [inline controls/roles/manage-service/tasks/write-grafana-configuration.yml:17]: see comment at line 10

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: controls/roles/manage-service/tasks/write-grafana-configuration.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    mode: 0655
+  become: yes
+  when: stereum_service_configuration.command.find("GRAFANA_INI_LH") != -1
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/tasks/write-grafana-configuration.yml:20]: Here the `when` is used to determine which consensys client is choosen. The `when` on the `main.yaml` is for grafana service itself

File: controls/roles/manage-service/tasks/write-grafana-configuration.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+- name: Copy Grafana's dashboard(s)
+  copy:
+    src: "{{ role_path }}/files/grafana/provisioning-lh/dashboards/BeaconProcessing.json"
+    dest: "{{ grafana_config_path }}/provisioning/dashboards"
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/tasks/write-grafana-configuration.yml]: done!

File: controls/roles/manage-service/tasks/write-grafana-configuration.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    mode: 0655
+  become: yes
+  when: stereum_service_configuration.command.find("GRAFANA_INI_LH") != -1
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/tasks/write-grafana-configuration.yml:28]: done!

File: controls/roles/manage-service/tasks/write-grafana-configuration.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
+---
+- name: Create directories for Grafana
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0655
+  loop:
+    - "{{ grafana_config_path }}/provisioning/datasources"
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/tasks/write-grafana-configuration.yml]: done!

File: controls/roles/manage-service/tasks/write-grafana-configuration.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/tasks/write-grafana-configuration.yml]: done!

File: controls/roles/manage-service/molecule/monitor-lighthouse/verify.yml [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+  - name: Node exporter
+    command: "docker logs --tail=100 stereum-ef7d66cc-9022-11ec-8bd0-57e35985136a"
+    register: node_exporter
+  #  lighthouse beacon logs
+  - name: Lighthouse beacon
+    command: "docker logs --tail=200 stereum-cbf9f518-8ac7-11ec-8da2-ebc33b8d2c86"
+    register: lighthouse_beacon
+  - assert:
+      that:
+      - grafana.stdout.find("HTTP Server Listen") != -1
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/molecule/monitor-lighthouse/verify.yml:62]: done!

**Review by stefa2k — APPROVED** [review]
lgtm, great work!

**Pull request merged**